src/nIsForName.js

```diff
@@ -21,8 +21,14 @@
  *
  * @return {string[]}
  */
-function nIsForName(names) {
-  // write code here
+function nIsForName(arrayOfName = []) {
+  const hasNFirst = arrayOfName.reduce((hasNFirst, person) => {
```

Comment:
you have shadowed variable, it is bad if the name of the local variable is the same as a variable in upper scope you lose access to it and it hard to understand with which variable you work

Reply:
about `hasNFirst` declared twice

Reply:
maybe there will be better and simpler to use another more specific method?